gravity.sh

```diff
@@ -598,7 +598,9 @@ gravity_DownloadBlocklistFromUrl() {
 
   # Check for allowed protocols
   if [[ $url != "http"* && $url != "https"* && $url != "file"* && $url != "ftp"* && $url != "ftps"* && $url != "sftp"* ]]; then
-    echo -e "${OVER}  ${CROSS} ${str} Invalid protocol specified, ignoring list"
+    echo -e "${OVER}  ${CROSS} ${str} Invalid protocol specified. Ignoring list."
+    echo -e "      Ensure your URL starts with a valid protocol like http:// or https:// ."
+    echo -e "      E.g., try http://localhost/adblock.txt instead of localhost/adblock.txt."
```

Comment:
```suggestion

```

I would remove this line, esp. because this seems to be an example of a locally hosted file. Instead users could use `file://` if they don't run a local webserver to serve the file.

Reply:
```suggestion
    echo -e "${OVER}  ${CROSS} ${str} Invalid protocol specified. Ignoring list."
    echo -e "      Ensure your URL starts with a valid protocol like http:// , https:// or file:// ."
```
Maybe something like that? I want to be clear for users that use local webservers and also for users who don't.